crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -205,7 +205,12 @@ impl JwtAuthHttpProxy {
     ) -> Result<JwtAuthHttpProxy, SMCumulocityMapperError> {
         let c8y_host = tedge_config.query_string(C8yUrlSetting)?;
         let device_id = tedge_config.query_string(DeviceIdSetting)?;
-        let http_con = reqwest::ClientBuilder::new().build()?;
+        let cert = std::fs::read(tedge_config.query(C8yRootCertPathSetting)?)?;
+        let cert_pem = reqwest::Certificate::from_pem(&cert)?;
```

Comment:
No need to check if the `cert` is a file or a directory, like it's done in the rest of the code?